### DISCUSSION TITLE
0.4.1h

### DISCUSSION THREAD

**Discussion by alienatedsec** [alienatedsec/yi-hack-v5#345] [Announcements]
## Breaking changes !!!
* There is a new baseline `0.4.1h`, so you need to prepare your SD card for it - follow https://github.com/alienatedsec/yi-hack-v5/discussions/267
* The camera web page is no longer under port `8080` but back to default port `80` - access your camera web page using its IP only. Those with proxy environments or `Home Assistant` integrations must amend their settings.
* The new baseline is needed to fix OTA upgrade #326 - cameras on earlier versions cannot upgrade from the WebUi as there is an additional baseline check - it refers to `0.4.1` for the latest release - https://github.com/alienatedsec/yi-hack-v5/commit/23a96a5700bb9616fc56b905e772737ed44870c6

* Future non-baseline upgrades will be released without SD formatted to the FAT32 requirement. 
* We need testers, so if you have no physical access to your camera - don't upgrade, as it may require fixing by downgrading etc.

## What's Changed
* Reduced `busybox` size to address #312
* Moved some tools from `rootfs` to `/tmp/sd/yi-hack-v5/.../[s]bin` folders
* Fixed Curl SSL crypto library
* First try to address #333 

## What is still not working
* #336 - unable to find the right `sleep` settings as suggested by @roleoroleo - please test and suggest if possible - it does not work on my cameras whatever I try - `yi_home_1080p - 48US` and `yi_dome - 43US`
* #333 - Please, I need mqtt tested by the community

**Full Changelog**: https://github.com/alienatedsec/yi-hack-v5/compare/0.4.1g...0.4.1h

<hr /><em>This discussion was created from the release <a href='https://github.com/alienatedsec/yi-hack-v5/releases/tag/0.4.1h'>0.4.1h</a>.</em>

**Comment by Minims** [comment]
@alienatedsec no HTTP and no SSH  after reboot for this release ? I will dig a bit more tomorrow. But both camera are like from factory. Cloud is working, but no access.

```
❯ nmap -P0 192.168.1.54
Starting Nmap 7.93 ( https://nmap.org ) at 2024-02-12 23:27 CET
Nmap scan report for 192.168.1.54
Host is up (0.0095s latency).
Not shown: 998 closed tcp ports (conn-refused)
PORT   STATE SERVICE
21/tcp open  ftp
23/tcp open  telnet

Nmap done: 1 IP address (1 host up) scanned in 0.25 seconds

```

**Reply by alienatedsec** [reply]
It could mean that it does not recognise the `/tmp/sd/yi-hack-v5` folder.

**Reply by Minims** [reply]
Connect through port 23 is OK. I have make a mistake while copying files... It's a bit late here :-).
Sorry for the noise. I will continue testing the new release.

**Reply by alienatedsec** [reply]
> Sorry for the noise. I will continue testing the new release.

@Minims I appreciate any help, even if it's a noise 😉

**Comment by Minims** [comment]
Missing a link to the binary on the sdcard ?

```
/home/yi-hack-v5 # vi
vi: applet not found
/home/yi-hack-v5 #
```

This one is OK.

```
/tmp/sd/yi-hack-v5 # ./bin/vi
```

**Reply by alienatedsec** [reply]
I thought I fixed it but must have done a reset before committing and building this release. The symlink needs to be deleted from the /bin folder and it will call the one on the SD card. 

```
rm /bin/vi
```
